plugins/DuskVerb/tools/tuner/staged_tuner.py

```
@@ -0,0 +1,1327 @@
+    if range_overrides:
+        ranges.update(range_overrides)
+    print(f"\n══════════ STAGE: {stage_name} ══════════")
+    print(f"  Active params: {sorted(active_params)}")
+    print(f"  Trials: {n_trials}   Workers: {workers}   Sampler: CMA-ES")
+    print(f"  Locked params: {len(locked_overrides)} fixed values")
+    sigma0 = 0.2
+    if sigma_scale:
+        s = max(sigma_scale.values())
+        if s > 1.01:
+            sigma0 = min(0.5, 0.2 * s)
+            print(f"  Sigma scaled:  {sigma0:.3f} (×{s:.2f} from base 0.2 — "
```

Ruff (0.15.15)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)